workshops/website_mockup_generator/README.md

```diff
@@ -0,0 +1,480 @@
+In order to change the background color and the text alignment inside the `body`, specify a class called `body` in the body tag. We will be providing CSS stylings for the `body` class later.
+
+```html
+<body class="body">
+```
+
+#### The heading
+In the `<body>` we will add a heading inside `<h1>...</h1>` tags.
+
+```html
+<h1>Website Mockup Generator</h1>
+```
+
+#### Placing a text input field
+In order to place a text input field to get the website from the URL, use the following line to create a text input with a placeholder 'Enter the website' and provide id to the input as 'websiteAddress'.
+
+```html
+<input type="text" id="websiteAddress" placeholder='Enter the website'>
+```
+
+#### Placing the buttons
+We need two buttons to generate the mockup and to download respectively. Use the `<button>` tag twice to place the buttons. Provide the button label in between the `<button></button>` tags. Provide the id as `submitButton` for the 'Generate mockup' button and `Download` for 'Download' button.
+
+```html
+<button type="submit" id="submitBtn" onclick='generateMockup();'>Generate Mockup</button>
+<button id="Download">Download</button>
+```
+
+#### Generated mockup image
+We will be using an `<img>` tag to display the generated mockup image. Provide the id as `resultImg`.
+
+```html
+<img id='resultImg'>
+```
+
+
+#### Final index.html code:
+
+```html
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>Website Mockup Generator</title>
+    <link href="style.css" rel="stylesheet" type="text/css" />
+  </head>
+  <body class="body">
+    <h1>Website Mockup Generator</h1>
+    <input type="text" id="websiteAdress" placeholder='Enter the website'>
+    <br><br>
+    <button type="submit" id="submitBtn" onclick='generateMockup();'>Generate Mockup</button>
+    <button id="Download">Download</button>
+    <img id='resultImg'>
+    <script src="script.js"></script>
+  </body>
+</html>
+```
+
+And that's it! We have completed writing our HTML. Your mockup generator should be looking similar to this :
+
+![Mockup generator with only HTML](https://cloud-cg4k9mn15.vercel.app/0mockupgenerator-html.png)
+
+
+### 2) style.css
+
+The style.css file is used to add styles to our website. You can optionally use your own CSS to make our site look better according to your wish!
+
+#### Adding custom font
+Import a custom font from Google fonts using the below code :
+
+```css
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600&display=swap");
+* {
+  font-family: "Poppins", sans-serif;
+}
+```
+
+#### Customizing the `<body>` style
+Now, let us customize the body style. Change the body background using `background-color: red;`. Change the body text color into white using `color : white;`. To make the website look better, let us align the items using `align-items: center;`. Finally, the CSS class for the body will be similar :

```

Comment:
```suggestion
Now, let us customize the `<body>` tag. we will change the background color to and the text color to white. To make the website look better, let us align our items to center using `align-items` and `justify-content`:
```

Reply:
this way, it will look better so that the person first reads what we will be doing and then actually implements it